lib/rules/unit-no-unknown/index.js

```diff
@@ -42,9 +43,15 @@ const rule = function(actual, options) {
       value = value.replace(/\*/g, ",");
       valueParser(value).walk(function(valueNode) {
         // Ignore wrong units within `url` function
+        // and within functions listed in the `ignoreFunctions` option
         if (
           valueNode.type === "function" &&
-          valueNode.value.toLowerCase() === "url"
+          (valueNode.value.toLowerCase() === "url" ||
+            optionsMatches(
+              options,
+              "ignoreFunctions",
+              valueNode.value.toLowerCase()
```

Comment:
```suggestion
              valueNode.value
```

I don't think we want to lowercase this as it makes the case insensitivity flag (`i`) redundant.